docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/local.md

```diff
@@ -1,16 +1,18 @@
 ---
-title: "Local agents"
+title: "Running a Dagster+ agent locally"
 displayed_sidebar: "dagsterPlus"
 sidebar_position: 40
 sidebar_label: "Local"
 ---
 
-You can set up a local agent to execute pipelines launched in Dagster+. This ensures that your user code is always executed on infrastructure you own. When a pipeline is executed, the local agent will create subprocesses to execute user code. However, it's important to keep in mind that  the local agent's ability to run user code is limited by the capacity of the machine on which it's running.
+You can set up a local agent to execute pipelines launched in Dagster+. Local agents are a good way to experiment with Dagster+ locally before deploying a more scalable hybrid agent like [Kubernetes](dagster-plus/hybrid/agents/kubernetes) or [Amazon ECS](dagster-plus/hybrid/agents/amazon-ecs-new-vpc).
 
-Running a local agent can be a good choice if:
-- The expected load of your pipelines is relatively light (guaranteed to fit on a single node).
-- Your pipelines aren't computationally intensive and don't use much memory.
-- You can restart the agent when you update your code.
+:::warning
+Local agents aren't well suited for most production use cases. This is because local agents:
+- Don't have the same CI / CD update strategies as the other agents.
```

Comment:
i also dont like the blocks on blocks on blocks look so this fixes two poblems!

Reply:
this was wording from me -- I am 💯  fine keeping the language here simple, but like our CI / CD templates work for the other hybrid agent types but not for local agents, correct? More for my own knowledge than for the docs at this point.

Reply:
Our CI/CD templates generally rely on slinging around container images. So as long as you configure the "local agent" (again, kind of a muddying of concepts - since we usually align "the way the agent is run" with "the way code servers are launched") to work with one of the containerized user code launchers, you could get our CI/CD templates to work with it.